### PR TITLE
Container inspect

### PR DESCRIPTION
**Pull request by jnummelin**
Feature to inspect a container through CLI. Helps debugging container issues and eliminates need to swim to nodes with SSH to inspect a container.

### CODE REVIEW

File: agent/lib/kontena/rpc/docker_container_api.rb [hunk 1]
```diff
@@ -147,6 +147,14 @@ def delete(id, opts = {})
         raise RpcServer::Error.new(404, 'Not found')
       end
 
+      ##
+      # @param [String] id
+      #
+      def inspect(id, opts)
```
Comment (jakolehm) [inline agent/lib/kontena/rpc/docker_container_api.rb]: opts is not used, do we need it?

File: cli/lib/kontena/cli/containers/inspect_command.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+    parameter "CONTAINER_ID", "Container id"
+
+    def execute
+      require_api_url
+      token = require_token
+
+      service_name = container_id.match(/(.+)-(\d+)/)[1]
+      result = client(token).get("containers/#{current_grid}/#{service_name}/#{container_id}/inspect")
+      puts puts JSON.pretty_generate(result)
```
Comment (jakolehm) [inline cli/lib/kontena/cli/containers/inspect_command.rb]: double puts

**Comment by jnummelin** [comment]
I fixed the unnecessary puts and opts thingies. :) I still have to figure why the branch has conflicts as on my end even rebase says that everything is up to date. :/

**Comment by nevalla** [comment]
Did you already try this: https://help.github.com/articles/syncing-a-fork/

**Comment by jakolehm** [comment]
I would like to see at least api level test for this

**Pull request merged**